src/MycilaTrafficLight.cpp

```diff
@@ -54,6 +54,29 @@ void Mycila::TrafficLight::begin(const int8_t greenPin, const int8_t yellowPin,
   }
 }
 
+#ifdef MYCILA_NEOPIXEL_SUPPORT
+void Mycila::TrafficLight::begin(const int16_t neoPin, const uint16_t neoCount, const neoPixelType neoType) {
+  // In case we already have a NeoPixel object, delete it
+  if (_strip != nullptr) {
+    delete _strip;
+    _strip = nullptr;
+  }
```

Comment:
this lib is the only one which didn't have a `_enabled` flag to check its lifecycle. I think it should have one now. I will add it in main in the same way I use to have it in all my libs so that it stays consistant.